submissions/emmanuelEsuong/javaScript/script.js

```diff
@@ -22,3 +22,23 @@ var productList = [
 for (let x = 0; x < productList.length; x++) {
     console.log(productList[x].productName + " with " + productList[x].imageUrl + " is sold for " + productList[x].productPrice + ".");
 }
+
+
+// 2nd Task
+
+var x = 123-7;
+
+for(var i = 1; i < x; i++){
```

Comment:
Since you are counting from `1`, make `i <= x`. Else, start your count from `0`

Reply:
Updated. Thanks.